internal/server/resp/server.go

```diff
@@ -214,7 +214,7 @@ func (s *Server) AcceptConnectionRequests(ctx context.Context, wg *sync.WaitGrou
 
 			handler := commandhandler.NewCommandHandler(cmdHandlerID, responseChan, preprocessingChan,
 				s.cmdWatchSubscriptionChan, parser, s.shardManager, s.globalErrorChan,
-				ioThreadReadChan, ioThreadWriteChan, ioThreadErrChan, s.wl)
+				ioThreadReadChan, ioThreadWriteChan, ioThreadErrChan, s.wl, false)
```

Comment:
this additional arg seems unnecessary after we make the changes I suggested above.